apps/client/src/features/app-settings/panel-content/PanelContent.module.scss

```
@@ -14,9 +9,29 @@
   position: relative;
 }
 
+.corner {
+  flex: 0 0 auto;
+  display: flex;
+  justify-content: flex-end;
+  // width is needed so the button can align to the right of the capped
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)